app/src/main/java/org/greenstand/android/TreeTracker/userselect/UserSelectScreen.kt

```diff
@@ -0,0 +1,38 @@
+package org.greenstand.android.TreeTracker.userselect
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import org.greenstand.android.TreeTracker.view.TreeTrackerTheme
+
+@Composable
+fun UserSelectScreen(viewModel: UserSelectViewModel, navController: NavController) {
+    TreeTrackerTheme {
```

Comment:
The `TreeTrackerTheme` was already applied in the `UserSelectFragment` so this shouldn't be necessary here. I had issues with applying the same composable twice that led to crashes but that was due to a doubly created `Scaffold` composable. I'm not sure if a doubly set `TreeTrackerTheme` would cause issues but it's technically not needed here